tests/php/test_class.jetpack.php

```diff
@@ -858,7 +853,20 @@ function test_verify_xml_rpc_signature() {
 		$this->assertSame( $user_id, $verified['user_id'], 'Expected user id to match with supplied user id.' );
 
 		// Cleanup.
-		Jetpack_Options::delete_option( array( 'user_tokens', 'master_user' ) );
+		Jetpack_Options::delete_option( 'user_tokens' );
+		$_SERVER['REQUEST_URI'] = '';
```

Comment:
This (and the lines that follow) look like something that would already exist. Oh, did you get this from the `WP_Test_Jetpack_REST_API_Authentication` tear-down? Regardless, I think it would make more sense having all cleanup in the existing `tearDown()` call.

Reply:
`tearDown()` runs after every test. Do we need to reset all these things every time?
The `unset()` could probably be pared down a little though

Reply:
> tearDown() runs after every test. Do we need to reset all these things every time?

I thought we did, but my phpunit-fu is _very_ weak. 🙃

Reply:
I was able to pear that down quite a bit. I think we should keep it within the test, given how no other test in the file needs these resets